src/Microsoft.Identity.Web.DownstreamApi/Microsoft.Identity.Web.DownstreamApi.csproj

```diff
@@ -1,13 +1,14 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <Title>Microsoft Identity Web downstream API</Title>
     <Product>Microsoft Identity Web downstream API</Product>
     <Description>Interface for higher level API for confidential client applications.</Description>
     <ProjectGuid>{A123BD94-812D-40EC-9576-1A7AB5C59913}</ProjectGuid>
     <PackageReadmeFile>README.md</PackageReadmeFile>
   </PropertyGroup>
-<PropertyGroup Condition=" '$(TargetFrameworkIdentifier)' == '.NETCoreApp' ">
```

Comment:
That was the big problem. If applies to net6.0 and net7.0, the code generator was silently crashing, and not generating for net8.0 and net9.0
cc: @keegan-caruso @GeoK @jennyf19